### ISSUE TITLE
Uberbar Search

### ISSUE THREAD

**Issue by dubrod** [modxcms/a11y#17]
For compliance with Web Content Accessibility Guidelines (WCAG)

Form label:

The search field needs a <label> tag, preferably one that is visible (the placeholder is not considered an accessible label because it disappears as soon as the keyboard focus enters the field), but you could also implement a hidden label in this particular case, because the magnifying glass conveys the idea of search (as long as the magnifying glass is turned into an actual submit button and as long as it has text that says "search").

Submit button is missing:

Ideally, the form would have an actual submit button so that blind users know they can submit the form. Blind users don't see the magnifying glass icon. In fact, the magnifying glass icon could be turned into a submit button, and you could maintain the same basic look as the exiting interface.

See accessibility to-do list at https://dequeuniversity.com/modx-accessibility.html#uberbar

**Comment by dubrod** [comment]
![image](https://cloud.githubusercontent.com/assets/595788/7235453/46ec567e-e75c-11e4-9f98-180b25b38731.png)

new search bar system put in.
